src/foam/nanos/menu/VerticalMenu.js

```diff
@@ -94,15 +96,17 @@ foam.CLASS({
         .addClass('side-nav-view')
         .start()
           .startContext({ data: this })
-            .start()
-              .add(self.MENU_SEARCH.clone().copyFrom({ view: {
-                class: 'foam.u2.view.TextField',
-                onKey: true
-              } }))
-              .addClass('foam-u2-search-TextSearchView')
-            .end()
+          .start()
+            .add(self.MENU_SEARCH.clone().copyFrom({ view: {
```

Comment:
Easier to just set the view: in the property itself instead of using copyFrom to set it here.

Reply:
fixed